ReactAndroid/src/main/java/com/facebook/react/views/androidtv/ReactAndroidTVRootViewHelper.java

```diff
@@ -0,0 +1,101 @@
+package com.facebook.react.views.androidtv;
+
+import android.view.KeyEvent;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.facebook.react.ReactRootView;
+import com.facebook.react.bridge.WritableMap;
+import com.facebook.react.bridge.WritableNativeMap;
+import com.facebook.react.modules.core.DeviceEventManagerModule.RCTDeviceEventEmitter;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class ReactAndroidTVRootViewHelper {
+
+  private static final List<Integer> PRESS_KEY_EVENTS = Arrays.asList(
+    KeyEvent.KEYCODE_DPAD_CENTER,
+    KeyEvent.KEYCODE_ENTER,
+    KeyEvent.KEYCODE_SPACE
+  );
+
+  private static final List<Integer> NAVIGATION_KEY_EVENTS = Arrays.asList(
+    KeyEvent.KEYCODE_DPAD_DOWN,
+    KeyEvent.KEYCODE_DPAD_LEFT,
+    KeyEvent.KEYCODE_DPAD_UP,
+    KeyEvent.KEYCODE_DPAD_RIGHT
+  );
+
+  private View mLastFocusedView = null;
+
+  private ReactRootView mReactRootView;
+
+  public ReactAndroidTVRootViewHelper(ReactRootView mReactRootView) {
```

Comment:
Rename argument to `reactRootView`, then you don't have to use `this.sth = sth` below